src/common/processing_linux.c

```diff
@@ -242,24 +243,25 @@ void ffProcessGetInfoLinux(pid_t pid, FFstrbuf* processName, FFstrbuf* exe, cons
 
     #elif defined(__sun)
 
-    char filePath[PATH_MAX];
-    snprintf(filePath, sizeof(filePath), "/proc/%d/psinfo", (int) pid);
+    char filePath[32];
+    snprintf(filePath, sizeof(filePath), "/proc/%d/", (int)pid);
+    FF_AUTO_CLOSE_FD int dfd = open(filePath, O_RDONLY | O_CLOEXEC);
```

Comment:
I don't like this

Reply:
All right, I'll remove this part, but the buf size will change to fit.

Reply:
Done.